conjur/cli.py

```diff
@@ -59,143 +63,154 @@ def copyright():
-            dest='ssl_verify',
-            action='store_false')
-
-        # The external names for these are unfortunately named so we remap them
-        global_optional.add_argument('-u', '--user', dest='login_id')
-        global_optional.add_argument('-k', '--api-key')
-        global_optional.add_argument('-p', '--password')
-
         global_optional.add_argument('-d', '--debug',
-            help='Enable debugging output',
-            action='store_true')
-
-        global_optional.add_argument('--verbose',
-            help='Enable verbose debugging output',
-            action='store_true')
+                                     help='Enable debugging output',
+                                     action='store_true')
 
+        global_optional.add_argument('--insecure',
+                                     help='Skip verification of server certificate (not recommended for production)',
+                                     dest='ssl_verify',
+                                     action='store_false')
 
         resource, args = Cli._parse_args(parser)
 
-        # We don't have a good "debug" vs "verbose" separation right now
-        if args.verbose is True:
-            args.debug = True
-
         # TODO Add tests for exception handling logic
+        # # pylint: disable=broad-except
         try:
+            self._setup_logging(args.debug)
+
             Cli.run_client_action(resource, args)
         except FileNotFoundError as not_found_error:
             sys.stdout.write(f"Error: No such file or directory: '{not_found_error.filename}'")
-            logging.info('Error: No such file or directory: %s', not_found_error.filename)
-            sys.exit(1)
         except requests.exceptions.HTTPError as http_error:
             sys.stdout.write(str(http_error))
-            logging.info(str(http_error))
-            sys.exit(1)
-        else:
-            sys.exit(0)
+        except ValueError as value_error:
+            sys.stdout.write(str(value_error))
+        except RuntimeError as runtime_error:
+            sys.stdout.write(str(runtime_error))
+        except Exception as general_exception:
+            sys.stdout.write(str(general_exception))
+
+        sys.exit(0)
 
     @staticmethod
     def run_client_action(resource, args):
         """
         Helper for creating the Client instance and invoking the appropriate
         api class method with the specified parameters.
         """
+        if resource == 'init':
+            init_input_obj = InputConjurrc(args.url,
+                                           args.account,
+                                           args.cert)
+            init_logic = InitLogic()
+
+            input_controller = InitController(init_input_obj,
+                                              init_logic,
+                                              args.force)
+            input_controller.call()
+            sys.exit(0)
```

Comment:
Needs a comment as to why sys exit is needed

Reply:
Added
'''
A successful exit is required to prevent the initialization of
the Client because the init command does not require the Client
'''